lib/rules/extends-before-declarations.js

```diff
@@ -13,25 +13,25 @@ module.exports = {
       var lastDeclaration = null;
 
       block.forEach(function (item, j) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Function 'anonymous' has a complexity of 6.](https://www.codacy.com/app/snugug/sass-lint/file/2320582947/issues/source?bid=3169912&fileBranchId=3169916#l15)